go.mod

```diff
@@ -3,16 +3,14 @@ module github.com/bmeg/grip
 go 1.12
 
 require (
-	github.com/AndreasBriese/bbloom v0.0.0-20190306092124-e2d15f34fcf9 // indirect
 	github.com/Shopify/sarama v1.22.1
-	github.com/bmeg/golib v0.0.0-20170626075926-82a1e1d7a0b2
+	github.com/bmeg/protoc-gen-grcp-rest-direct v0.0.0-20190228222353-4d40e8b9d305 // indirect
 	github.com/boltdb/bolt v1.3.1
+	github.com/ckaznocha/protoc-gen-lint v0.2.1 // indirect
 	github.com/davecgh/go-spew v1.1.1
-	github.com/dgraph-io/badger v2.0.0-rc2+incompatible
-	github.com/dgryski/go-farm v0.0.0-20190423205320-6a90982ecee2 // indirect
+	github.com/dgraph-io/badger v0.0.0-20190809121831-9d7b751e85c9
```

Comment:
Can this be linked to a named badger version?

Reply:
updated to v2.0.0